src/js/api/urls.js

```diff
@@ -30,6 +30,9 @@ export const STOCK_MOVEMENT_ITEM_REMOVE = id => `${STOCK_MOVEMENT_ITEM_API}/${id
 // STOCK TRANSFER
 export const STOCK_TRANSFER_API = `${API}/stockTransfers`;
 export const STOCK_TRANSFER_DELETE = id => `${STOCK_TRANSFER_API}/${id}`;
+export const STOCK_TRANSFER_UPDATE = id => `${STOCK_TRANSFER_API}/${id}`;
+export const STOCK_TRANSFER_GET = id => `${STOCK_TRANSFER_API}/${id}`;
```

Comment:
I apologize not for catching it earlier, but I think we used to name such urls as `..._BY_ID`, not to repeat the urls for update, delete, get, read, so I think it should be:
```js
export const STOCK_TRANSFER_BY_ID = id => `${STOCK_TRANSFER_API}/${id}`;
```

Reply:
Someone only did it for stock movements, I think it could be just `STOCK_TRANSFER`, and then you just have:
`delete(STOCK_TRANSFER)`, `update(STOCK_TRANSFER)` etc.

Reply:
It was done only for stock movements, because it was the moment in which we decided to avoid using `DELETE`, `GET`, `POST`, and `PUT`. I remember we created a ticket for the refactor of the rest of the namings (but I could be wrong).